### PR TITLE
remove ethersprovider

### LINKED ISSUE #1100: Remove `geth` from TEE ts-tests
### Context
We shouldn't need it, it's enough to generate the signature using js lib only.

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### PR DESCRIPTION
**Pull request by 0xverin**
deleted ethers provider,only kept the ethers account;
issue:#1100

### CODE REVIEW

File: tee-worker/ts-tests/identity.test.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ const ethereumIdentity = <LitentryIdentity>{
 
 const substrateIdentity = <LitentryIdentity>{
     handle: {
-        Address32: `0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d`, //alice
+        Address32: `0x91fe1e29ddfc3306c7f2c864fc5599b7a060ecf5c972b810b0c2f8e7d42cff77`, //alice
```
Comment (Kailai-Wang) [inline tee-worker/ts-tests/identity.test.ts]: This doesn't look like `//Alice` or `//alice`, what's this number?

File: tee-worker/ts-tests/identity.test.ts [hunk 2]
```diff
@@ -52,10 +52,7 @@ const ethereumValidationData = <LitentryValidationData>{
         Evm: {
             message: `0x${Buffer.from('mock_message', 'utf8').toString('hex')}`,
             signature: {
-                Ethereum: `0x${Buffer.from(
-                    '10ee76e356d944d17bce552a4fd0d4554ccc97dc81213f470367bd3b99c441c51',
-                    'utf8'
-                ).toString('hex')}`,
+                Ethereum: '' as HexString,
```
Comment (Kailai-Wang) [inline tee-worker/ts-tests/identity.test.ts:55]: may I ask why it's empty now? Then CI shouldn't pass I assume?

File: tee-worker/ts-tests/identity.test.ts [hunk 3]
```diff
@@ -65,10 +62,7 @@ const substrateValidationData = <LitentryValidationData>{
         Substrate: {
             message: `0x${Buffer.from('mock_message', 'utf8').toString('hex')}`,
             signature: {
-                Sr25519: `0x${Buffer.from(
-                    '10ee76e356d944d17bce552a4fd0d4554ccc97dc81213f470367bd3b99c441c51',
-                    'utf8'
-                ).toString('hex')}`,
+                Sr25519: '' as HexString,
```
Comment (Kailai-Wang) [inline tee-worker/ts-tests/identity.test.ts:65]: same as above

File: tee-worker/ts-tests/identity.test.ts (hunk 2 above)
Comment (0xverin) [reply tee-worker/ts-tests/identity.test.ts:55]: I think it's a `UnexpectedMessage` , so I set it to empty and update it here:https://github.com/litentry/litentry-parachain/blob/35ba8fdc692501851f040cff679f30ffa19e0baa/tee-worker/ts-tests/identity.test.ts#L152

**Review by ziming-zung — APPROVED** [review]
Looks good.

**Pull request merged**